src/sentry/models/organization.py

```diff
@@ -157,6 +159,13 @@ def get_organizations_where_user_is_owner(self, user_id: int) -> QuerySet:
         return self.filter(id__in=owner_team_member_orgs).union(owner_role_orgs)
 
 
+class InvalidRegionalSlugTargetException(Exception):
+    pass
+
+
+region_prefix_regex = r"^(r[-|_|\s][a-zA-Z]{2}[-|_|\s])+(.*)"
```

Comment:
```suggestion
region_prefix_regex = re.compile(r"^(r[-|_|\s][a-zA-Z]{2}[-|_|\s])+(.*)")
```

Could we pre-compile this pattern so that we don't have to re-compile it on each org save?